src/PKSim.UI/Views/Formulations/CreateFormulationView.Designer.cs

```diff
@@ -28,60 +28,6 @@ protected override void Dispose(bool disposing)
       /// </summary>
       private void InitializeComponent()
       {
-         ((System.ComponentModel.ISupportInitialize)(this.layoutControlBase)).BeginInit();
-         this.layoutControlBase.SuspendLayout();
-         ((System.ComponentModel.ISupportInitialize)(this.layoutControlGroupBase)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this.layoutItemOK)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this.layoutItemCancel)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this.emptySpaceItemBase)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this.layoutItemExtra)).BeginInit();
-         ((System.ComponentModel.ISupportInitialize)(this._errorProvider)).BeginInit();
-         this.SuspendLayout();
-         // 
-         // btnCancel
-         // 
-         this.btnCancel.Location = new System.Drawing.Point(516, 12);
-         this.btnCancel.Size = new System.Drawing.Size(106, 22);
-         // 
-         // btnOk
-         // 
-         this.btnOk.Location = new System.Drawing.Point(386, 12);
-         this.btnOk.Size = new System.Drawing.Size(126, 22);
-         // 
-         // layoutControlBase
-         // 
-         this.layoutControlBase.Location = new System.Drawing.Point(0, 676);
-         this.layoutControlBase.Size = new System.Drawing.Size(634, 46);
-         this.layoutControlBase.Controls.SetChildIndex(this.btnCancel, 0);
-         this.layoutControlBase.Controls.SetChildIndex(this.btnOk, 0);
```

Comment:
All this stuff is the real change. The rest was just reformatting
Basically I had to remove all the junks that was in the designer file :)